app/components/notification_inbox_component.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="row notification-inbox">
+  <div class="mx-auto">
+    <% @notifications.each do |notification| %>
```

Comment:
Just a question here: Can't we create the `NotificationComponent`s first and then render them all at once?

Reply:
I don’t get your question. What’s the problem with this approach?

Reply:
I thought that one render call might be better than multiple ones?